v2/internal/attacktechniques/aws/persistence/iam-create-backdoor-role/malicious_policy.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
```

Comment:
any reason why this first statement? I don't see it mentioned in the InvictusIR write-up

Reply:
No real reason, as I saw the issue saying that this technique should be similar to aws.persistence.iam-backdoor-role I used the same policy.

Reply:
I don't think so, backdoor-role starts from an existing role (e.g., one that would be used by an EC2 instance already). If you create a new role, I don't think you'd have a trust on EC2

Reply:
done in [f34d94e](https://github.com/DataDog/stratus-red-team/pull/478/commits/f34d94e613f24c53a8c4b2177a808b28d526a440)